webui/src/views/entries/dialogs/IncomeExpenseDialog.vue

```diff
@@ -29,6 +30,8 @@ const props = defineProps({
     visible: { type: Boolean, default: false }
 })
 
+const categoryId = ref()
```

Comment:
can you propagate the category id back to the entries view so that it will be part of the payload upon save/edit